src/Identity.Governance/Identity.Governance/custom/New-MgEntitlementManagementAccessPackageAssignmentRequest.ps1

```diff
@@ -0,0 +1,246 @@
+          if ($AccessPackageAssignmentId -ne $null -and $AccessPackageAssignmentId.Length -ne 0) {
+              $RequestType = "AdminRemove"
+          } else {
+              $RequestType = "AdminAdd"
+          }
+          write-debug "setting requesttype $RequestType"
+      }
+  
+      if ($RequestType -ne "AdminRemove") {
+         if ($null -ne $StartDate -or $StartDate.Length -eq 0) {
+              $now = Get-Date
+              $ts = Get-Date $now.ToUniversalTime() -format "s"
+              $StartDate = $ts + "Z"
+          }
+      }
+  
+      $AccessPackageAssignmentRequestBodyAccessPackageAssignment = new-object microsoft.graph.powershell.models.MicrosoftGraphAccessPackageAssignment
+      if ($AccessPackageAssignmentId -ne $null -and $AccessPackageAssignmentId.Length -ne 0) {
+          $AccessPackageAssignmentRequestBodyAccessPackageAssignment.Id = $AccessPackageAssignmentId
+      }
+      if ($TargetId -ne $null -and $TargetId.Length -ne 0) {
+          $AccessPackageAssignmentRequestBodyAccessPackageAssignment.TargetId = $TargetId
+      }
+      if ($AssignmentPolicyId -ne $null -and $AssignmentPolicyId.Length -ne 0) {
+          $AccessPackageAssignmentRequestBodyAccessPackageAssignment.AssignmentPolicyId = $AssignmentPolicyId
+      }
+      if ($AccessPackageId -ne $null -and $AccessPackageId.Length -ne 0) {
+          $AccessPackageAssignmentRequestBodyAccessPackageAssignment.AccessPackageId = $AccessPackageId
+      }
+      
+      if ($null -ne $StartDate -and $StartDate.Length -ne 0) {
+          $AccessPackageAssignmentRequestBodyAccessPackageAssignment.Schedule = new-object Microsoft.Graph.PowerShell.Models.MicrosoftGraphRequestSchedule
+          $AccessPackageAssignmentRequestBodyAccessPackageAssignment.Schedule.StartDateTime = $StartDate
+      }
+    
+      $null = $PSBoundParameters.Remove("AccessPackageAssignmentId")
+      $null = $PSBoundParameters.Remove("AccessPackageId")
+      $null = $PSBoundParameters.Remove("AssignmentPolicyId")
+      $null = $PSBoundParameters.Remove("TargetId")
+      $null = $PSBoundParameters.Remove("StartDate")
+
+      $PSBoundParameters['AccessPackageAssignment'] = $AccessPackageAssignmentRequestBodyAccessPackageAssignment
+
+      $PSBoundParameters['Answers'] = $Answers
+      if ($Justification -ne $null -and $Justification.Length -ne 0) {
+        $PSBoundParameters['Justification'] = $Justification
+      }
+      $PSBoundParameters['RequestType'] = $RequestType
+  
+      try {
+        $outBuffer = $null
+        if ($PSBoundParameters.TryGetValue('OutBuffer', [ref]$outBuffer)) {
+            $PSBoundParameters['OutBuffer'] = 1
+        }
+        $subecmd = 'Microsoft.Graph.Identity.Governance.private\New-MgEntitlementManagementAccessPackageAssignmentRequest_CreateExpanded'
+        
+        $subWrappedCmd = $ExecutionContext.InvokeCommand.GetCommand(($subecmd), [System.Management.Automation.CommandTypes]::Cmdlet)
```

Comment:
See my comments here https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/632.

Reply:
Updated to move to process section.